Packs/Yara/IndicatorFields/indicatorfield-Author.json

```diff
@@ -0,0 +1,28 @@
+{
+    "id": "indicator_author",
+    "version": -1,
+    "modified": "2024-03-18T15:43:53.640998055Z",
+    "name": "Author",
+    "ownerOnly": false,
+    "cliName": "author",
+    "type": "shortText",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedToAll": true,
```

Comment:
if this field associated to all, dont you think we should add the field to commontypes pack?

Reply:
Done